scripts/deploy-ebook-to-www

```diff
@@ -310,8 +310,11 @@ do
 			# We do this first because the tweaks below shouldn't apply to the single-page file
 			se recompose-epub --xhtml --output "${workDir}"/single-page.xhtml --extra-css-file="${webRoot}/css/web.css" "${workDir}"
 
+			# Wrap book contents in a <main> tag (and extra <div> for styling reasons)
```

Comment:
But one issue that using flexbox introduces is it removes the existing margin collapsing behavior of the body margins with the top + bottom section/image margins (https://stackoverflow.com/questions/43882869/margin-collapsing-in-flexbox). With the workaround, we get the old behavior back and everything should remain pixel identical with master excluding the new footer.